src/js/components/stock-movement-wizard/modals/SubstitutionsModal.jsx

```diff
@@ -264,12 +297,15 @@ class SubstitutionsModal extends Component {
         }
 
         this.setState({
+          emptySubstitutions: !substitutions.length,
+          loadingSubstitutions: false,
           formValues: {
             substitutions,
           },
           originalItem,
         });
-      }).catch(err => err);
+      })
+      .catch(err => err);
```

Comment:
don't we want to mark `loadingSubstitutions` as `true` if we get an error also? In my opinion if any error occurs, the spinner will still be visible.
You could use `.finally` in the end, so you don't have to hide the spinner twice